Gopkg.toml

```diff
@@ -53,7 +53,7 @@
 
 [[override]]
   name = "github.com/tendermint/iavl"
-  version = "=v0.9.2"
+  revision = "bab6ea5006ebf4e97e918eadcd6ea338e6d415eb"
```

Comment:
You can use `version = "=v0.10.0"` as soon as https://github.com/tendermint/iavl/pull/100 is merged and tagged on master.